<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="17" passed="12" failed="5" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-03-24T14:40:29 IST" name="Suite" finished-at="2025-03-24T14:41:44 IST" duration-ms="75238">
    <groups>
    </groups>
    <test started-at="2025-03-24T14:40:29 IST" name="parameterTest" finished-at="2025-03-24T14:41:44 IST" duration-ms="75238">
      <class name="testscript.SearchUserPageTest">
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:testscript.SearchUserPageTest@1890516e]" started-at="2025-03-24T14:41:02 IST" name="beforeMethod" finished-at="2025-03-24T14:41:05 IST" duration-ms="3059" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="verifyWhetherSearchResultsAreDisplayed()[pri:0, instance:testscript.SearchUserPageTest@1890516e]" started-at="2025-03-24T14:41:05 IST" name="verifyWhetherSearchResultsAreDisplayed" finished-at="2025-03-24T14:41:07 IST" duration-ms="2016" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyWhetherSearchResultsAreDisplayed -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:testscript.SearchUserPageTest@1890516e]" started-at="2025-03-24T14:41:07 IST" name="afterMethod" finished-at="2025-03-24T14:41:07 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- testscript.SearchUserPageTest -->
      <class name="testscript.AddNewSubCategoryPageTest">
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:testscript.AddNewSubCategoryPageTest@3336e6b6]" started-at="2025-03-24T14:41:26 IST" name="beforeMethod" finished-at="2025-03-24T14:41:29 IST" duration-ms="2522" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="verifyIfNewSubCategoryCanBeAdded()[pri:0, instance:testscript.AddNewSubCategoryPageTest@3336e6b6]" started-at="2025-03-24T14:41:29 IST" name="verifyIfNewSubCategoryCanBeAdded" finished-at="2025-03-24T14:41:32 IST" duration-ms="3136" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyIfNewSubCategoryCanBeAdded -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:testscript.AddNewSubCategoryPageTest@3336e6b6]" started-at="2025-03-24T14:41:32 IST" name="afterMethod" finished-at="2025-03-24T14:41:32 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- testscript.AddNewSubCategoryPageTest -->
      <class name="testscript.AddNewUserPageTest">
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:testscript.AddNewUserPageTest@7ba8c737]" started-at="2025-03-24T14:40:55 IST" name="beforeMethod" finished-at="2025-03-24T14:40:57 IST" duration-ms="2155" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="verifyIfNewUserCanBeAdded()[pri:0, instance:testscript.AddNewUserPageTest@7ba8c737]" started-at="2025-03-24T14:40:57 IST" name="verifyIfNewUserCanBeAdded" finished-at="2025-03-24T14:41:02 IST" duration-ms="4358" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyIfNewUserCanBeAdded -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:testscript.AddNewUserPageTest@7ba8c737]" started-at="2025-03-24T14:41:02 IST" name="afterMethod" finished-at="2025-03-24T14:41:02 IST" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- testscript.AddNewUserPageTest -->
      <class name="testscript.DeleteUserPageTest">
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:testscript.DeleteUserPageTest@15a04efb]" started-at="2025-03-24T14:41:07 IST" name="beforeMethod" finished-at="2025-03-24T14:41:09 IST" duration-ms="2480" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="verifyIfUserCanBeDeleted()[pri:0, instance:testscript.DeleteUserPageTest@15a04efb]" started-at="2025-03-24T14:41:09 IST" name="verifyIfUserCanBeDeleted" finished-at="2025-03-24T14:41:11 IST" duration-ms="1985" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyIfUserCanBeDeleted -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:testscript.DeleteUserPageTest@15a04efb]" started-at="2025-03-24T14:41:11 IST" name="afterMethod" finished-at="2025-03-24T14:41:11 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- testscript.DeleteUserPageTest -->
      <class name="testscript.HomePageTest">
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:testscript.HomePageTest@93081b6]" started-at="2025-03-24T14:40:51 IST" name="beforeMethod" finished-at="2025-03-24T14:40:53 IST" duration-ms="2498" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="verifyWhetherUsercanLogout()[pri:0, instance:testscript.HomePageTest@93081b6]" started-at="2025-03-24T14:40:53 IST" name="verifyWhetherUsercanLogout" finished-at="2025-03-24T14:40:55 IST" duration-ms="1864" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyWhetherUsercanLogout -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:testscript.HomePageTest@93081b6]" started-at="2025-03-24T14:40:55 IST" name="afterMethod" finished-at="2025-03-24T14:40:55 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- testscript.HomePageTest -->
      <class name="testscript.UpdateCategoryPageTest">
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:testscript.UpdateCategoryPageTest@26adfd2d]" started-at="2025-03-24T14:41:21 IST" name="beforeMethod" finished-at="2025-03-24T14:41:24 IST" duration-ms="2370" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="verifyIfUserCanEditCategoryPage()[pri:0, instance:testscript.UpdateCategoryPageTest@26adfd2d]" started-at="2025-03-24T14:41:24 IST" name="verifyIfUserCanEditCategoryPage" finished-at="2025-03-24T14:41:26 IST" duration-ms="2517" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyIfUserCanEditCategoryPage -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:testscript.UpdateCategoryPageTest@26adfd2d]" started-at="2025-03-24T14:41:26 IST" name="afterMethod" finished-at="2025-03-24T14:41:26 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- testscript.UpdateCategoryPageTest -->
      <class name="testscript.ResetPageTest">
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:testscript.ResetPageTest@16c069df]" started-at="2025-03-24T14:41:11 IST" name="beforeMethod" finished-at="2025-03-24T14:41:14 IST" duration-ms="2537" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="verifyWhetherClickOnResetButtonisResettingfields()[pri:0, instance:testscript.ResetPageTest@16c069df]" started-at="2025-03-24T14:41:14 IST" name="verifyWhetherClickOnResetButtonisResettingfields" finished-at="2025-03-24T14:41:16 IST" duration-ms="1964" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyWhetherClickOnResetButtonisResettingfields -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:testscript.ResetPageTest@16c069df]" started-at="2025-03-24T14:41:16 IST" name="afterMethod" finished-at="2025-03-24T14:41:16 IST" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- testscript.ResetPageTest -->
      <class name="testscript.UpdateSubCategoryPageTest">
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:testscript.UpdateSubCategoryPageTest@7c3fdb62]" started-at="2025-03-24T14:41:32 IST" name="beforeMethod" finished-at="2025-03-24T14:41:34 IST" duration-ms="2091" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="verifyIfUserCanUpdateSubCategory()[pri:0, instance:testscript.UpdateSubCategoryPageTest@7c3fdb62]" started-at="2025-03-24T14:41:34 IST" name="verifyIfUserCanUpdateSubCategory" finished-at="2025-03-24T14:41:37 IST" duration-ms="2782" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyIfUserCanUpdateSubCategory -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:testscript.UpdateSubCategoryPageTest@7c3fdb62]" started-at="2025-03-24T14:41:37 IST" name="afterMethod" finished-at="2025-03-24T14:41:37 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- testscript.UpdateSubCategoryPageTest -->
      <class name="testscript.LoginPageTest">
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:testscript.LoginPageTest@264f218]" started-at="2025-03-24T14:40:29 IST" name="beforeMethod" finished-at="2025-03-24T14:40:31 IST" duration-ms="1998" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="verifyUserCanLoginwithValidUsernameAndInvalidPwd(java.lang.String,java.lang.String)[pri:0, instance:testscript.LoginPageTest@264f218]" started-at="2025-03-24T14:40:31 IST" name="verifyUserCanLoginwithValidUsernameAndInvalidPwd" finished-at="2025-03-24T14:40:31 IST" duration-ms="364" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[admin1]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[pass]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Login is not Successful expected [Dashboard | 7rmart supermarket] but found [Login | 7rmart supermarket]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Login is not Successful expected [Dashboard | 7rmart supermarket] but found [Login | 7rmart supermarket]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at testscript.LoginPageTest.verifyUserCanLoginwithValidUsernameAndInvalidPwd(LoginPageTest.java:54)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyUserCanLoginwithValidUsernameAndInvalidPwd -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:testscript.LoginPageTest@264f218]" started-at="2025-03-24T14:40:31 IST" name="afterMethod" finished-at="2025-03-24T14:40:31 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:testscript.LoginPageTest@264f218]" started-at="2025-03-24T14:40:31 IST" name="beforeMethod" finished-at="2025-03-24T14:40:33 IST" duration-ms="1660" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="verifyUsercanLoginwithInvalidUsernameAndPassword(java.lang.String,java.lang.String)[pri:0, instance:testscript.LoginPageTest@264f218]" started-at="2025-03-24T14:40:33 IST" name="verifyUsercanLoginwithInvalidUsernameAndPassword" data-provider="credentials" finished-at="2025-03-24T14:40:33 IST" duration-ms="466" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[admin]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[admin]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyUsercanLoginwithInvalidUsernameAndPassword -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:testscript.LoginPageTest@264f218]" started-at="2025-03-24T14:40:33 IST" name="afterMethod" finished-at="2025-03-24T14:40:33 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:testscript.LoginPageTest@264f218]" started-at="2025-03-24T14:40:33 IST" name="beforeMethod" finished-at="2025-03-24T14:40:35 IST" duration-ms="1602" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="verifyUsercanLoginwithInvalidUsernameAndPassword(java.lang.String,java.lang.String)[pri:0, instance:testscript.LoginPageTest@264f218]" started-at="2025-03-24T14:40:35 IST" name="verifyUsercanLoginwithInvalidUsernameAndPassword" data-provider="credentials" finished-at="2025-03-24T14:40:35 IST" duration-ms="386" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[123]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[admin]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Login is not Successful expected [Dashboard | 7rmart supermarket] but found [Login | 7rmart supermarket]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Login is not Successful expected [Dashboard | 7rmart supermarket] but found [Login | 7rmart supermarket]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at testscript.LoginPageTest.verifyUsercanLoginwithInvalidUsernameAndPassword(LoginPageTest.java:66)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyUsercanLoginwithInvalidUsernameAndPassword -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:testscript.LoginPageTest@264f218]" started-at="2025-03-24T14:40:35 IST" name="afterMethod" finished-at="2025-03-24T14:40:35 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:testscript.LoginPageTest@264f218]" started-at="2025-03-24T14:40:35 IST" name="beforeMethod" finished-at="2025-03-24T14:40:37 IST" duration-ms="1807" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="verifyUsercanLoginwithInvalidUsernameAndPassword(java.lang.String,java.lang.String)[pri:0, instance:testscript.LoginPageTest@264f218]" started-at="2025-03-24T14:40:37 IST" name="verifyUsercanLoginwithInvalidUsernameAndPassword" data-provider="credentials" finished-at="2025-03-24T14:40:38 IST" duration-ms="590" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[admin]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[123]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Login is not Successful expected [Dashboard | 7rmart supermarket] but found [Login | 7rmart supermarket]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Login is not Successful expected [Dashboard | 7rmart supermarket] but found [Login | 7rmart supermarket]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at testscript.LoginPageTest.verifyUsercanLoginwithInvalidUsernameAndPassword(LoginPageTest.java:66)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyUsercanLoginwithInvalidUsernameAndPassword -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:testscript.LoginPageTest@264f218]" started-at="2025-03-24T14:40:38 IST" name="afterMethod" finished-at="2025-03-24T14:40:38 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:testscript.LoginPageTest@264f218]" started-at="2025-03-24T14:40:38 IST" name="beforeMethod" finished-at="2025-03-24T14:40:40 IST" duration-ms="2029" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="verifyUsercanLoginwithInvalidUsernameAndPassword(java.lang.String,java.lang.String)[pri:0, instance:testscript.LoginPageTest@264f218]" started-at="2025-03-24T14:40:40 IST" name="verifyUsercanLoginwithInvalidUsernameAndPassword" data-provider="credentials" finished-at="2025-03-24T14:40:41 IST" duration-ms="847" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[123]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[123]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Login is not Successful expected [Dashboard | 7rmart supermarket] but found [Login | 7rmart supermarket]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Login is not Successful expected [Dashboard | 7rmart supermarket] but found [Login | 7rmart supermarket]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at testscript.LoginPageTest.verifyUsercanLoginwithInvalidUsernameAndPassword(LoginPageTest.java:66)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyUsercanLoginwithInvalidUsernameAndPassword -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:testscript.LoginPageTest@264f218]" started-at="2025-03-24T14:40:41 IST" name="afterMethod" finished-at="2025-03-24T14:40:41 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:testscript.LoginPageTest@264f218]" started-at="2025-03-24T14:40:41 IST" name="beforeMethod" finished-at="2025-03-24T14:40:43 IST" duration-ms="2799" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="verifyUsercanLoginwithInvalidUsernameAndValidPwd()[pri:0, instance:testscript.LoginPageTest@264f218]" started-at="2025-03-24T14:40:43 IST" name="verifyUsercanLoginwithInvalidUsernameAndValidPwd" finished-at="2025-03-24T14:40:47 IST" duration-ms="3587" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Login is not Successful expected [Dashboard | 7rmart supermarket] but found [Login | 7rmart supermarket]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Login is not Successful expected [Dashboard | 7rmart supermarket] but found [Login | 7rmart supermarket]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at testscript.LoginPageTest.verifyUsercanLoginwithInvalidUsernameAndValidPwd(LoginPageTest.java:42)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyUsercanLoginwithInvalidUsernameAndValidPwd -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:testscript.LoginPageTest@264f218]" started-at="2025-03-24T14:40:47 IST" name="afterMethod" finished-at="2025-03-24T14:40:47 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:testscript.LoginPageTest@264f218]" started-at="2025-03-24T14:40:47 IST" name="beforeMethod" finished-at="2025-03-24T14:40:50 IST" duration-ms="2880" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="verifyUsercanLoginwithValidCredentials()[pri:0, instance:testscript.LoginPageTest@264f218]" started-at="2025-03-24T14:40:50 IST" name="verifyUsercanLoginwithValidCredentials" finished-at="2025-03-24T14:40:51 IST" duration-ms="963" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyUsercanLoginwithValidCredentials -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:testscript.LoginPageTest@264f218]" started-at="2025-03-24T14:40:51 IST" name="afterMethod" finished-at="2025-03-24T14:40:51 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- testscript.LoginPageTest -->
      <class name="testscript.UpdateContactUsPageTest">
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:testscript.UpdateContactUsPageTest@205d38da]" started-at="2025-03-24T14:41:37 IST" name="beforeMethod" finished-at="2025-03-24T14:41:40 IST" duration-ms="2830" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="verifyIfUserCanEditContactDetails()[pri:0, instance:testscript.UpdateContactUsPageTest@205d38da]" started-at="2025-03-24T14:41:40 IST" name="verifyIfUserCanEditContactDetails" finished-at="2025-03-24T14:41:44 IST" duration-ms="4300" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyIfUserCanEditContactDetails -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:testscript.UpdateContactUsPageTest@205d38da]" started-at="2025-03-24T14:41:44 IST" name="afterMethod" finished-at="2025-03-24T14:41:44 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- testscript.UpdateContactUsPageTest -->
      <class name="testscript.AddCategoryPageTest">
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:testscript.AddCategoryPageTest@2bec854f]" started-at="2025-03-24T14:41:16 IST" name="beforeMethod" finished-at="2025-03-24T14:41:18 IST" duration-ms="2150" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="verifyUsercanAddCategory()[pri:0, instance:testscript.AddCategoryPageTest@2bec854f]" started-at="2025-03-24T14:41:18 IST" name="verifyUsercanAddCategory" finished-at="2025-03-24T14:41:21 IST" duration-ms="3507" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyUsercanAddCategory -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:testscript.AddCategoryPageTest@2bec854f]" started-at="2025-03-24T14:41:21 IST" name="afterMethod" finished-at="2025-03-24T14:41:21 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- testscript.AddCategoryPageTest -->
    </test> <!-- parameterTest -->
  </suite> <!-- Suite -->
</testng-results>
